src/screens/ThreeDsProcessors/ThreeDsProcessorHome.res

```diff
@@ -272,6 +272,8 @@ let make = () => {
               connector=connectorName
               setScreenState={_ => ()}
               isPayoutFlow=false
+              setCurrentStep={_ => ()}
```

Comment:
can we make this optional parameter as it won't be much work

Reply:
we will  remove the whole quick start module.